Assets/SteamVR_Unity_Toolkit/Scripts/VRTK_InteractableObject.cs

```diff
@@ -95,6 +106,19 @@ public enum AllowedController
         private bool previousIsGrabbable;
         private bool forcedDropped;
 
+        public bool CheckHideMode(bool default_mode, ControllerHideMode override_mode )
```

Comment:
All public methods require documenting in the relevant section (check the ControllerEvents section for an example of how class methods are documented)